content/news/2024-10-30-maplibre-newsletter-october-2024/index.md

```diff
@@ -29,43 +29,53 @@ Here's an interesting situation that should be possible now:
 
 We have introduced some breaking changes in these versions as was planned for version 5, so make sure to take a look at the changelog for more details.
 There are still open issues with globe and other improvements we plan to add as part of this major release. Stay tuned!
 See the initial post in the following issue to see the current status:
 
-[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834] (https://github.com/maplibre/maplibre-gl-js/issues/3834)
+[Breaking Changes in MapLibre GLJS v5 maplibre gl-js#3834](https://github.com/maplibre/maplibre-gl-js/issues/3834)
 
 Overall we are progressing well and ironing out the globe feature.
 
 ## MapLibre Native
 
-- Development of the Vulkan backend continued this month. We also ran another benchmark on AWS Device Farm, using even more device types this time. One conclusion you may draw from the results is that Vulkan will offer a solid performance boost on modern devices. We're still validating these results. Stay tuned for an Android official release following later this year!
+For MapLibre Native, we have continued development of Vulkan, along with Android and IoS updates and feature parity. 
 
+- Development of the Vulkan backend continued this month. We also ran another benchmark on AWS Device Farm, using even more device types this time. One conclusion you may draw from the results is that Vulkan will offer a solid performance boost on modern devices. We're still validating these results. Stay tuned for an Android official release following later this year!
 - Kaushal Kumar Singh rewrote the Android build config from Groovy to the more modern Kotlin in [#2902](https://github.com/maplibre/maplibre-native/pull/2902).
-
 -Support for text-variable-anchor-offset has been merged [#2921](hhttps://github.com/maplibre/maplibre-native/pull/2921). This feature is one of the features that Microsoft championed to be added to MapLibre GL JS, and now ported to MapLibre Native. We're happy with this contribution and are excited about Microsoft adopting MapLibre Native.
-
-- MapLibre iOS [6.7.1 and 6.8.0] were released. (https://github.com/maplibre/maplibre-native/releases?q=ios&expanded=true).
-
+- MapLibre iOS [6.7.1 and 6.8.0](https://github.com/maplibre/maplibre-native/releases?q=ios&expanded=true) were released.
 - MapLibre Android [11.5.1] was released. (https://github.com/maplibre/maplibre-native/releases?q=android-v11&expanded=true).
-
 - We also made two releases for [MapLibre Android v10](https://github.com/maplibre/maplibre-native/releases?q=android-v10&expanded=true) with some backports.
 
 ## Flutter
 
 We’re conducting a brief survey to gather feedback on MapLibre Flutter. The survey takes only about 2 minutes to complete and will help to identify strengths and areas for improvement. If you've worked with MapLibre on Flutter, your input is highly appreciated.
-[You can participate here](https://forms.gle/UTB6KjVFtmYv2RzU8)
+You can participate [here](https://forms.gle/UTB6KjVFtmYv2RzU8)
```

Comment:
Missing dot at the end of the sentence.

Reply:
Added after "here" otherwise I see this grammatically fine